src/services/TitleService.js

```diff
@@ -0,0 +1,13 @@
+import Service from '../abc/Service';
+export default class TitleService extends Service {
+	constructor(app, serviceName="TitleService"){
+		super(app, serviceName);
+		this.App = app;
+		this.title = this.App.props.configdefaults?.title;
```

Comment:
@Aringoaway You dont want to take the title from `configdefaults` but from `config`. The best option is to use `this.App.Config.get("title")`